requirements.md

```diff
@@ -91,6 +114,8 @@ authorization it can be validated against relying-party stored instrument displa
 
 * For privacy, the protocol must enable the user to require that two payment instruments NOT be associated with the same authenticator.
 
+Q: Please clarify.
```

Comment:
I think this bullet point is meant to say 'with the same credential' rather than authenticator, but I would also still like to understand the privacy concern in that case better as well :D

Reply:
Make sense. I think the relationship between the instruments and credentials needs to be clearly defined because it's reflected on the API.

Reply:
I have removed this requirement. The goal was to not reveal through the protocol itself that, for example, the user of a personal card is the same user that has a different business card. So far I don't see the API doing that, but we should keep an eye out for it. I will resolve this conversation since I've removed the (unclear) requirement. I welcome a clearer proposal.